SQL/updates/53-54.sql

```diff
@@ -0,0 +1,9 @@
+# Updating DB from 23-24,
```

Comment:
You either mistyped on the numpad or grabbed a REALLY old file

Reply:
Yeah, that's a result of me using sabre's old file, oops! I'll fix that